patternlab/styleguide/source/_patterns/01-atoms/09-media/figure--dataviz.twig

```diff
@@ -0,0 +1,31 @@
+{% if figure.dataviz %}
+  {# Render figure.content, which will be the main content of the figure #}
+  {# Important: we wrap this in an `if` statement to allow consuming systems
+     to either use figure.image or figure.content (if they prefer to externally
+     render figure.content
+   #}
+  {% set align = figure.align|default('left') %}
+  {% set id = figure.dataviz.id %}
+  {# TODO:  SET SIZE VALUES #}
+  {# Set default size to large. #}
+  {% set size = "ma__figure--" ~ figure.dataviz.size|default('large') %}
+  {% set class = size ~ ' ' ~ 'ma__dataviz' %}
+  {% set content %}
+    <div id="{{ figure.dataviz.id }}"></div>
```

Comment:
Can you see a reason that we would want a static ID here?  We just need it to be able to reference the div a moment later, and it seems like needing to generate a unique ID might be a pain for consuming systems.  I'd probably recommend that we randomly generate one.  Eg: 
```
{% set id = 'dataviz-' ~ random(9999) %}
```

Reply:
Implemented in #971.

Reply:
I thought the id value passed from Drupal would be unique enough. So, I set it as static.  That's not the case?